docs/index.html

```diff
@@ -96,6 +96,7 @@ <h2>Configuration</h2>
             <span class="hljs-attr">hookRenderRoute</span>: <span class="hljs-literal">true</span>, <span class="hljs-comment">// Activate the prune during the `hook:render:route`</span>
             <span class="hljs-attr">hookGeneratePage</span>: <span class="hljs-literal">true</span>, <span class="hljs-comment">// Activate the prune during the `hook:generate:page`</span>
             <span class="hljs-attr">lighthouseUserAgent</span>: <span class="hljs-string">&apos;lighthouse&apos;</span>, <span class="hljs-comment">// Value of the Lighthouse UserAgent, either as String or RegExp (a string will be converted to a case-insensitive RegExp in the MobileDetect library)</span>
+            <span class="hljs-attr">headerOverrideName</span>: <span class="hljs-string">&apos;user-agent&apos;</span>, <span class="hljs-comment">// Value of a custom header name passed from a Lambda Edge function, or similar</span>
```

Comment:
This file is automatically generated during the _nuxt-generate_ command: `npm run generate` (it's missing from the documentation, i'm gonna update it).

BTW: It's ok 🚀